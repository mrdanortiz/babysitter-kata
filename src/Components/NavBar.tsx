import React, { FunctionComponent } from "react";
import { Container, Menu } from "semantic-ui-react";

export const NavBar: FunctionComponent = () => (
    <Menu inverted={true} data-testid="navbar">
        <Container>
            <Menu.Item header={true}>
                Babysitter Kata
            </Menu.Item>
            <Menu.Item href="https://github.com/mrdanortiz/babysitter-kata" target="_blank">
                Source Code
            </Menu.Item>
        </Container>
    </Menu>
);

export default NavBar;
