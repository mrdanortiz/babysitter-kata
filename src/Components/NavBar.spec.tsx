import * as React from "react";
import { render } from "react-testing-library";
import { NavBar } from "./NavBar";

it("renders NavBar without crashing", () => {

    const { getByTestId } = render(<NavBar />);
    expect(getByTestId("navbar")).toMatchSnapshot();

});
