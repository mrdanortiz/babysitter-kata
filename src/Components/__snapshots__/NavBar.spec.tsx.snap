// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavBar without crashing 1`] = `
<div
  data-testid="navbar"
  class="ui inverted menu"
>
  <div
    class="ui container"
  >
    <div
      class="header item"
    >
      Babysitter Kata
    </div>
    <a
      href="https://github.com/mrdanortiz/babysitter-kata"
      target="_blank"
      class="item"
    >
      Source Code
    </a>
  </div>
</div>
`;
